movement/log_config.py

```diff
@@ -0,0 +1,65 @@
+import logging
+from logging.handlers import RotatingFileHandler
+from pathlib import Path
+from typing import Optional
+
+from rich.logging import RichHandler
+
+FORMAT = "%(asctime)s - %(levelname)s - "
+FORMAT += "%(processName)s %(filename)s:%(lineno)s - %(message)s"
+
+
+def configure_logging(
+    log_level: int = logging.INFO,
```

Comment:
If the logging is only to file, I would set this to `DEBUG`. There's no reason to only send some of the logs to file.